### PR TITLE
docs: add mutual links to concept and manifest pages

### PR DESCRIPTION
**Pull request by toricls**
This PR adds mutual links to the concept and manifest pages to make it easy to reference for users (especially for people visit our docs via search engines).

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: site/content/docs/concepts/pipelines.en.md [hunk 1]
```diff
@@ -89,6 +89,7 @@ stages:
       name: prod
       # requires_approval: true
 ```
+To learn about the specification of `pipeline.yml`, see the [pipeline manifest](../manifest/pipeline.en.md) page.
```
Comment (bvtujo) [inline site/content/docs/concepts/pipelines.en.md]:
```suggestion
You can see every available configuration option for `pipeline.yml` on the [pipeline manifest](../manifest/pipeline.en.md) page.
```

File: site/content/docs/manifest/backend-service.en.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-List of all available properties for a `'Backend Service'` manifest.
+List of all available properties for a `'Backend Service'` manifest. To learn about its concept, see the [Services](../concepts/services.en.md) page.
```
Comment (bvtujo) [inline site/content/docs/manifest/backend-service.en.md]:
```suggestion
List of all available properties for a `'Backend Service'` manifest. To learn about Copilot services, see the [Services](../concepts/services.en.md) concept page.
```

File: site/content/docs/manifest/lb-web-service.en.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-List of all available properties for a `'Load Balanced Web Service'` manifest.
+List of all available properties for a `'Load Balanced Web Service'` manifest. To learn about its concept, see the [Services](../concepts/services.en.md) page.
```
Comment (bvtujo) [inline site/content/docs/manifest/lb-web-service.en.md]:
```suggestion
List of all available properties for a `'Load Balanced Web Service'` manifest. To learn about Copilot services, see the [Services](../concepts/services.en.md) concept page.
```

File: site/content/docs/manifest/pipeline.en.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-List of all available properties for a Copilot pipeline manifest.
+List of all available properties for a Copilot pipeline manifest. To learn about its concept, see the [Pipelines](../concepts/pipelines.en.md) page.
```
Comment (bvtujo) [inline site/content/docs/manifest/pipeline.en.md]:
```suggestion
List of all available properties for a Copilot pipeline manifest. To learn more about pipelines, see the [Pipelines](../concepts/pipelines.en.md) concept page.
```

File: site/content/docs/manifest/scheduled-job.en.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-List of all available properties for a `'Scheduled Job'` manifest.
+List of all available properties for a `'Scheduled Job'` manifest. To learn about its concept, see the [Jobs](../concepts/jobs.en.md) page.
```
Comment (bvtujo) [inline site/content/docs/manifest/scheduled-job.en.md]:
```suggestion
List of all available properties for a `'Scheduled Job'` manifest. To learn about Copilot jobs, see the [Jobs](../concepts/jobs.en.md) concept page.
```

**Review by bvtujo — COMMENTED** [review]
Thank you, Tori! I just have some suggestions on the phrasing.

**Comment by toricls** [comment]
Thank you @bvtujo, I updated them! 🙌

**Review by bvtujo — APPROVED** [review]
:shipit:

**Review by iamhopaul123 — APPROVED** [review]
This is awesome! Thank you so much for contributing ❤️

**Pull request merged**